CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 ## [UNRELEASED] - YYYY-MM-DD
 ### Fixed 
-- Fixed a bug where appending data would not be correctly displayed in the history ([#446]) https://github.com/cbrnr/mnelab/pull/446 by [bkloeckl] (https://github.com/bkloeckl))
+- Fix a bug where appending data would not be correctly displayed in the history ([#446]) https://github.com/cbrnr/mnelab/pull/446 by [Benedikt Kloeckl](https://github.com/bkloeckl))
+
+### Added
+- Add tests for model.append_data() by generating temporary .edf files by [Benedikt Kloeckl](https://github.com/bkloeckl)).
```

Comment:
This is a user-facing document, so I would not mention any changes to tests here.